apps/system/js/app_chrome.js

```diff
@@ -199,6 +202,7 @@
     this.menuButton = this.element.querySelector('.menu-button');
     this.windowsButton = this.element.querySelector('.windows-button');
     this.title = this.element.querySelector('.chrome-title-container > .title');
+    this.siteIcon = this.element.querySelector('.site-icon');
```

Comment:
while waiting for bug 1168944, people are still going to tap the new shiny icon.
currently event fluffling will often trigger the back button when you do that :/

to prevent this I'd like to add an empty click event listener on the `siteIcon` as part of this patch to be filled up later on.